### ISSUE TITLE
Does not work with fastboot

### ISSUE THREAD

**Issue by iamareebjamal** [Cropster/ember-l10n#59]
Starting from 4.0.0, it does not work with fastboot

```
ReferenceError: document is not defined
    at Class.<anonymous> (/home/iamareebjamal/git/open-event-frontend/dist/assets/addon-tree-output/ember-l10n/services/l10n.js:184:1)
```

**Comment by mydea** [comment]
Hmm good catch. Probably needs adaptions akin to these:
https://github.com/adopted-ember-addons/ember-cli-ifa/pull/57/files

I will see if I find some time to work on this!

**Comment by mydea** [comment]
I have looked into it a bit, and got it running with Fastboot. You can try the branch `fastboot` - let me know if there are any issues with it!

**Comment by mydea** [comment]
It is merged and will soon be released as 4.1.0!
